src/compute_pair_local.cpp

```diff
@@ -264,11 +267,20 @@ int ComputePairLocal::compute_pairs(int flag)
         if (nvalues == 1) ptr = &vlocal[m];
         else ptr = alocal[m];
 
+        // to make sure dx, dy and dz are always from the lower to the higher id
+        double directionCorrection = i > j ? -1.0 : 1.0;
```

Comment:
Shouldn't this be using `itag` and `jtag`?

Reply:
Thanks! As noted in the implementation notes, I was not sure about this, but makes sense. Thanks, fixed in 74577fa.